src/utils/usageData.ts

```diff
@@ -165,11 +178,48 @@ const sendEvent = (category, action, label) => {
     );
 };
 
+/**
+ * Send usage data event to Google Analytics
+ * @param {EventAction} action The event action
+ * @param {string} label The event label
+ * @returns {void}
+ */
+const sendUsageData = <T>(action: T, label: string): void => {
+    if (!isInitialized) {
+        eventQueue.push({ action, label });
+        return;
+    }
+    while (eventQueue.length) {
+        const event = eventQueue.shift() as EventAction<T>;
+        sendEvent(
+            appJson.name,
+            (event.action as unknown) as string,
+            event.label || ''
+        );
+    }
+    sendEvent(appJson.name, action, label || '');
+};
+
+/**
+ * Send error usage data event to Google Analytics and also show it in the logger view
+ * @param {EventAction} error The event action
```

Comment:
As mentioned in the chat, I would rather we avoid using this type of documentation since we are using Typescript anyway. If we really want to use it however, the parameter here is wrong.

Reply:
Right, this was written before. I will fix the doc and will probably remove all the docs later.